src/test/java/com/getindata/kafka/connect/iceberg/sink/TestIcebergUtil.java

```diff
@@ -57,7 +57,7 @@ public void testUnwrapJsonRecord() throws IOException {
                 MAPPER.readTree(unwrapWithSchema).get("payload"), null,
                 MAPPER.readTree(unwrapWithSchema).get("schema"), null);
         Schema schema = e.icebergSchema();
-        GenericRecord record = e.asIcebergRecord(schema);
+        GenericRecord record = e.asIcebergRecord(schema, "__source_ts_ms");
```

Comment:
could be extracted as constant

Reply:
unsure where to put it, would be best to grab the default configuration setting. i guess it doesn't make much of a difference for the tests anyway :)

Reply:
I mean in that class as private static final to avoid having literal three times.